spec/forms/curation_concerns/multi_volume_work_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe CurationConcerns::MultiVolumeWorkForm do
+  subject { described_class.new(work, nil) }
+  let(:work) { MultiVolumeWork.new }
+
+  it "doesn't initialize description" do
```

Comment:
This shouldn't stop that. It just makes it not fill a new record with a default of an empty array.